orttraining/orttraining/python/training/ortmodule/_experimental/debug.py

```diff
@@ -0,0 +1,74 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# debug.py
+
+import os
+from . import _load_config_from_json as loading_functions
+
+def load_from_json(ortmodule, path=None):
+    """Load config from json file at given path.
+
+    Here is the schema that the json file must adhere to:
+    {
+        "PropagateCastOps":
+        {
+            "Strategy": "FLOOD_FILL", # str representing strategy (like "NONE", "FLOOD_FILL"...)
+            "Level": 3, # propagate cast ops level as an int
+            "Allow": ["ABC", "DEF"] # propagate cast ops allow as list of strs
+        },
+        "UseExternalGPUAllocator" : false, # bool flag
+        "EnableCustomAutogradFunction": true, # bool flag
+        "AllowLayerNormModPrecision": true, # bool flag
+        "EnableGradAccOptimization": true, # bool flag
+        "UseStaticShape": true, # bool flag
+        "RunSymbolicShapeInference": false, # bool flag
+        "SkipCheck": # list of strs representing `_SkipCheck`s checks to skip which will be aggregated using |
+        [
+            "SKIP_CHECK_DEVICE",
+            "SKIP_CHECK_BUILD_GRADIENT",
+            "SKIP_CHECK_EXECUTION_AGENT"
+        ],
+        "DebugOptions": # debug options for user facing configuration
+        {
+            "LogLevel": "VERBOSE",
+            "SaveONNX": true,
+            "ONNXPrefix": "my_model",
+            "SaveONNXPath": "/path/to/onnx/directory"
+        }
+    }
+
+    Args:
+        ortmodule (:obj:`ORTModule`): ORTModule instance that needs to be configured
+        path (:obj:`str`, optional): Path to json file. Alternatively, users can set the
+            environement variable ORTMODULE_JSON_CONFIG_PATH to the json config path. In case
+            both path and environment variable are set, the environment variable gets precedence.
+    """
+
+    json_path_environment_key = "ORTMODULE_JSON_CONFIG_PATH"
```

Comment:
That probably should be a global as opposed to implementation detail

Reply:
yeah, let me make this variable global.